e2e/cypress/integration/messaging/long_draft_spec.js

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Messaging', () => {
+    before(() => {
+        // # Make sure the viewport is the expected one, so written lines always create new lines
+        cy.viewport(1000, 660);
+
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M18699-Leave a long draft in the main input box', () => {
+        const lines = ['Lorem ipsum dolor sit amet,',
+            '\nconsectetur adipiscing elit.',
+            '\nNulla ac consectetur quam.',
+            '\nPhasellus libero lorem,',
+            '\nfacilisis in purus sed, auctor.'];
+
+        // # Create new DM channel with user's email
+        cy.apiGetUsers(['user-1', 'sysadmin']).then((userResponse) => {
+            const userEmailArray = [userResponse.body[1].id, userResponse.body[0].id];
+            cy.wrap(lines.reduce((prev, cur) => prev + cur, '')).as('fullText');
+
+            cy.apiCreateDirectChannel(userEmailArray).then(() => {
+                cy.visit('/ad-1/messages/@sysadmin');
+
+                // # Get the height before starting to write
+                cy.get('#post_textbox', {timeout: TIMEOUTS.LARGE}).type('a').clear();
+                cy.get('#post_textbox').then((post) => {
+                    cy.wrap(parseInt(post[0].clientHeight, 10)).as('previousHeight');
+                });
+
+                // # Post first line to use
+                cy.get('#post_textbox').type(lines[0]);
+
+                // # For each line
+                for (var i = 1; i < lines.length; i++) {
+                    // # Post the line
+                    cy.get('#post_textbox').type(lines[i]);
```

Comment:
Having [newline prefix](https://github.com/mattermost/mattermost-webapp/pull/4193/files#diff-f121c84c1ea2dda0f3e91119ee32f1bbR24-R27) (`\n...`) is less like a user action.  Should we pretend like a user is typing by removing it and doing like below?
```javascript
cy.get('#post_textbox').type('{shift}{enter}').type(lines[i]);
```